src/test/scala/beam/utils/MathUtilsSpec.scala

```diff
@@ -0,0 +1,38 @@
+package beam.utils
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+import scala.util.Random
+
+/**
+  * @author Dmitry Openkov
+  */
+class MathUtilsSpec extends AnyWordSpecLike with Matchers {
+  "round uniformly" when {
+    "receive a positive double value" should {
+      "round it to the closest integer most of the time (proportionally to the fraction)" in {
```

Comment:
Is this built strong enough such that it won't cause flaky tests? Random makes for some hard to control tests...

Reply:
It's not going to cause flaky tests because it uses seeding and a strict sequence of random numbers.